old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -212,52 +231,56 @@ the UTM zone, to avoid negative Easting numbers.
 
 <figure>
     <a href="https://www.e-education.psu.edu/natureofgeoinfo/sites/www.e-education.psu.edu.natureofgeoinfo/files/image/utm_zone_characteristics.png">
-    <img src="https://www.e-education.psu.edu/natureofgeoinfo/sites/www.e-education.psu.edu.natureofgeoinfo/files/image/utm_zone_characteristics.png">
+    <img src="https://www.e-education.psu.edu/natureofgeoinfo/sites/www.e-education.psu.edu.natureofgeoinfo/files/image/utm_zone_characteristics.png" alt="Two UTM zones, one north and one south. The zones are triangular and become more narrow towards the poles. The origin is at the south west corner of each zone which makes it on the equator for the north zones and near the tip of the triangle for the south zones.">
     </a>
-    <figcaption>The 0,0 origin of each UTM zone is located in the **Bottom left** hand corner (south west) of the zone - exactly 500,000 m EAST from the central meridian of the zone.  
+    <figcaption>The 0,0 origin of each UTM zone is located in the **bottom left** hand corner (south west) of the zone - exactly 500,000 m EAST from the central meridian of the zone.  
     Source: Penn State E-education</figcaption>
 </figure>
 
 ***
 
 ### More on UTM
 
-* <a href="https://www.e-education.psu.edu/geog482spring2/c2_p22.html" target="_blank">
+* <a href="https://www.e-education.psu.edu/natureofgeoinfo/c2_p22.html" target="_blank">
 Penn State E-education overview of UTM</a>
-* <a href="https://www.e-education.psu.edu/geog482spring2/c2_p23.html
-" target="_blank">
+* <a href="https://www.e-education.psu.edu/natureofgeoinfo/c2_p23.html" target="_blank">
 More about UTM Zones - Penn State E-education</a>
 
 ***
 
 <figure>
-    <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ed/Utm-zones.jpg/800px-Utm-zones.jpg">
-    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ed/Utm-zones.jpg/800px-Utm-zones.jpg">
+	<a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/geospatial-skills/gridded-Utm-zones.jpg">
+	<img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/geospatial-skills/gridded-Utm-zones.jpg" alt="A world map with a grid overlay. The grid is labeled 1 to 60 going east and C to X going north.">
```

Comment:
Could you please be a bit more specific about what this grid is showing? These are the UTM zone grids for the earth based on the full zone notation. (Not to be confused with the 'north' and 'south' notation as discussed in [this great Wiki article](https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system#Notation))